x-pack/legacy/plugins/siem/public/components/inspect/modal.test.tsx

```diff
@@ -111,19 +116,118 @@ describe('Modal Inspect', () => {
 
       expect(
         wrapper
-          .find('.euiCodeBlock')
+          .find('.euiDescriptionList__title')
           .first()
           .text()
-      ).toBe('My request');
+      ).toBe('Index pattern');
+      expect(
+        wrapper
+          .find('.euiDescriptionList__description')
+          .first()
+          .text()
+      ).toBe('auditbeat-*, filebeat-*, packetbeat-*, winlogbeat-*');
+      expect(
+        wrapper
+          .find('.euiDescriptionList__title')
+          .at(1)
```

Comment:
Consider (when possible) replacing class selectors specified in calls to `.find` (e.g. `.find('.euiDescriptionList__title')` with specific `data-test-subj` attribute selectors, because the latter can eliminate the need for calls like `at(1)`, which can be brittle if the order of elements on the page changes.

For example, if the code tested by this assertion:

```
      title: (
        <>
          {i18n.QUERY_TIME}
          <EuiIconTip content={i18n.QUERY_TIME_DESC} position="top" type="iInCircle" />
        </>
      ),
```

includes a `data-test-subj` attribute:

```
      title: (
        <span data-test-subj="query-time-label">
          {i18n.QUERY_TIME}
          <EuiIconTip content={i18n.QUERY_TIME_DESC} position="top" type="iInCircle" />
        </span>
      ),
```

the `at(1)` can be eliminated:

```
        wrapper
          .find('[data-test-subj="query-time-label"]')
          .first()
          .text()
```